cli/configCra.js

```diff
@@ -69,11 +69,11 @@ try {
       )
       // Require `env` direct from `react-scripts`
       .replace(
-        `require('../config/env');`,
-        `require('react-scripts/config/env');`,
+        "require('../config/env');",
+        "require('react-scripts/config/env');",
```

Comment:
Can you tell me which rule of eslint is this?

Reply:
Revert all linting changes for now